api/Makefile

```diff
@@ -1,13 +1,22 @@
-PROTO_PATH=/usr/local/include/
+.PHONY: all
+all: go_pipelinespec go_cachekey
 
-gen_pipeline_spec_proto: v2alpha1/pipeline_spec.proto
-	cd v2alpha1 && protoc -I=$(PROTO_PATH) -I=. \
-		--go_out=go \
+.PHONY: clean
+clean:
+	rm -rf v2alpha1/go
+
+go_pipelinespec: v2alpha1/pipeline_spec.proto
+	go install google.golang.org/protobuf/cmd/protoc-gen-go
+	mkdir -p v2alpha1/go/pipelinespec
+	cd v2alpha1 && protoc -I=. \
+		--go_out=go/pipelinespec \
 		--go_opt=paths=source_relative \
 		pipeline_spec.proto
 
-gen_cache_key_proto: v2alpha1/cache_key.proto
-	cd v2alpha1 && protoc -I=$(PROTO_PATH) -I=. \
-		--go_out=go \
+go_cachekey: v2alpha1/pipeline_spec.proto v2alpha1/cache_key.proto
+	go install google.golang.org/protobuf/cmd/protoc-gen-go
+	mkdir -p v2alpha1/go/cachekey
+	cd v2alpha1 && protoc -I=. \
+		--go_out=go/cachekey \
```

Comment:
note, cache_key is used internally, so it's put to a different folder from pipeline spec.